CS483/CS483/Kartaclysm/StateMachine/Gameplay/StateRacing.cpp

```diff
@@ -100,6 +98,13 @@ void Kartaclysm::StateRacing::BeginRace()
 	HeatStroke::GameObject* pTrack = m_pGameObjectManager->CreateGameObject(m_mContextParams.at("TrackDefinitionFile"), "Track");
 	ComponentTrack* pTrackComponent = static_cast<ComponentTrack*>(pTrack->GetComponent("GOC_Track"));
 
+	// TODO: weird issue with Racer transforms unless m_vWorldTranslation is changed on walls
```

Comment:
I don't understand what's happening here.  What's the weird issue with Racer transforms? And you say it happens unless m_vWorldTranslation is changed on walls, but you're only ever calling GetTranslation(), never changing it.  Is this because GetTranslation() does something with the transform hierarchy? Should that be handled as part of creating an object instead?